src/components/CreateWatch.svelte

```diff
@@ -0,0 +1,75 @@
+<script>
+import { getActiveFilterKeys } from '../data/filtering'
+import { getFiltersForWatch, getWatchableKeys } from '../data/watch'
+import { create } from '../data/watch'
+import { createdWatch } from '../data/analytics'
+import FilterTag from './FilterTag.svelte'
+import jquery from 'jquery' // $ is already a reserved token in Svelte (stores)
+import { onMount } from 'svelte'
+
+export let filter
+
+let name = ''
+let submitted = false
+
+function onSubmit() {
+  create(name, filter)
+  createdWatch()
+  submitted = true
+}
+
+onMount(async () => {
```

Comment:
no need for `async` here.  this looks like a different technique than was used over in the `Login.svelte`.  If we like this one better let's update that one over there.  I think it would be ideal to stick with one technique if possible.

Reply:
I have no opinion. Didn't even recognize the two were related other than use of jquery.